### DISCUSSION TITLE
The next release (6.2.58) and betas

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1614] [Release works]
Hi @everyone,
We'd be better to release new versikon, maybe 6.2.58, in September.

Before that, several betas might be released: 6.2.57 beta 1 would be released around June 20th.  Is it ok?

**Comment by racke** [comment]
Sounds good to me if you apply s/June/July/. Any specific plans for new features in 6.2.58?

**Comment by ikedas** [comment]
He he, it's July.
There are not so remarkable features.
- sympa-community/sympa#924 : The first step to unify list config and site/robot config.  Check `doc/sympa_config.5` for description about unified config.
- sympa-community/sympa#955 : Deprecate implicit sync_include.  By this change, inclusion will occur only by explicit operation or periodical sync_include tasks; Sudden implicit inclusion will no longer occur.
- For the other changes see [milestone 6.2.58](https://github.com/sympa-community/sympa/milestone/14).

**Comment by ikedas** [comment]
Additionally, following bugs would be better to be solved before the next stable.
- sympa-community/sympa#969; Included owner cannot be privileged owner
- sympa-community/sympa#466; Problem caused by comma in source_parameters

**Comment by ikedas** [comment]
Done!

**Comment by ikedas** [comment]
I'd like to release **one more beta**, because some fixes after 6.2.57b.1 have not been tested.  I think ---
- 6.2.57b.2 will be released in 22 or 23 Sep.
- 6.2.58 will be postponed for a few weeks.

**Comment by ikedas** [comment]
Done!

**Comment by ikedas** [comment]
6.2.58 will be released in **Tuesday, 20 October**.

- Fixes of CVE reported in sympa-community/sympa#1009 _won't_ be included in 6.2.58. --- In the first place, this issue is the same as sympa-community/sympa#946. As far as I have seen, rapid progress may not be expected.
- In above, sympa-community/sympa#466 hasn't been fixed.  Though it need small change of Sympa::Config, I couldn't have the time to do it.

**Comment by racke** [comment]
Please take a look at https://github.com/sympa-community/sympa/pull/1015 for issue #946.

**Comment by ikedas** [comment]
Hi @everyone,
Milestone 6.2.58 is getting to close.  No more fixes/enhancements should be added to 6.2.58.  The other open issues/PRs currently sugmitted may be assigned to the future milestones.

**Comment by ikedas** [comment]
Done!
